undead-discord.css

```diff
@@ -24,7 +24,7 @@
+.theme-dark .themed-popout .footer, .messages-wrapper, .chat > .content, .account, .channel-members, .scrollbar, .scrollbar .thumb, .scrollbar::after, .divider-red span, .search-bar, .typing, .connecting, .layer .content-region, .layer .sidebar-region, .layer .pad, .ui-button.filled:hover, .ui-checkbox.checked, .connect-account-btn-inner, .ui-tab-bar-item:active, .connect-account-btn-inner:hover, .modal-container .markdown-modal, .modal-container .markdown-modal-footer, .modal-container .markdown-modal-header, .modal-container .scroller-wrap, .modal-container .scrollbar .track, .popout-menu-item:hover, .themed-popout, .messages-popout-wrap .has-more button:hover, .mentioned .message-text, #user-profile-modal .scroller, .note textarea:focus, #user-profile-modal .empty, .context-menu, #user-profile-modal .btn.add-friend, .messages-wrapper .scrollbar .track, .context-menu .item-subMenu, .form-inner, .form-header, .form-actions, .form .btn-default, .form .btn-primary:hover, .popout, .channel-textarea-autocomplete-inner, .btn-item:hover, .private-channels, .messages .divider:not(.red) span, #friends, .friends-header, .friends-table, .friends-action, .private-channel-recipients-invite button, .upload-modal, .upload-modal .footer, .upload-modal .button, .upload-modal .button-primary:hover, .emoji-picker .scrollbar .track, .emoji-picker, .emoji-picker .sticky-header, .create-guild-container .join:hover .btn, .create-guild-container, .search-results-wrap, .search-results-wrap .channel-name, .emoji-picker .thumb, .private-channels .track, #friends .btn:hover, .add-friend-input-wrapper, .react-datepicker__day--disabled, .react-datepicker__day--outside-month, .Select-option.is-focused, .instant-invite-modal, .invite-button-icon, .deprecated-settings-modal .settings-inner, .deprecated-settings-modal .settings-actions, .ui-popout-list .ui-selectable-item:hover, .ui-standard-sidebar-view .btn-close, .checkbox-inner span, .slider-handle span, .region-select button:hover, .region-select-modal, .region-select-modal-option:hover, .jump-button, #rtc-connection, #rtc-connection-popout section, .emoji-picker .pad, .search-popout, .channel-textarea-guard button:hover, .message-group-blocked, .channels-wrap > div, .buttonBrandFilledDefault-2Rs6u5:hover, .scrollbar-2so3Ly, .pad-3D5tOh, .track-2Kex6E, .checked-2TahvT, .switch-3lyafC::before, .input-2YozMi, .message-group > .comment > div:not(.message), #user-profile-modal .track, .modal-3HOjGZ, .buttonRedFilledDefault-1TrZ9q:hover, .footer-1PYmcw, .selectorSelected-2M0IGv, .headerNormal-1cioxU, .footer-2J5zqP, .quickswitcher {
```

Comment:
I don't think the `.theme-dark` is ever necessary. Would probably be easier to just use `.themed-popout .footer`.